samples/Azure.Management.Storage/Generated/BlobContainerOperations.cs

```diff
@@ -5,14 +5,51 @@
 
 #nullable disable
 
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.ResourceManager.Core;
+
 namespace Azure.Management.Storage
 {
     /// <summary> A class representing the operations that can be performed over a specific BlobContainer. </summary>
-    public partial class BlobContainerOperations
+    public partial class BlobContainerOperations : ResourceOperationsBase<BlobContainer>
     {
         /// <summary> Initializes a new instance of BlobContainerOperations for mocking. </summary>
         protected BlobContainerOperations()
         {
         }
+
+        private static readonly ResourceType ResourceType = "Azure.Management.Storage/BlobContainerOperations";
+        protected override ResourceType ValidResourceType => ResourceType;
+
+        /// <inheritdoc />
+        public override ArmResponse<BlobContainer> Get(CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <inheritdoc />
+        public override Task<ArmResponse<BlobContainer>> GetAsync(CancellationToken cancellationToken = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary> Lists all available geo-locations. </summary>
+        /// <returns> A collection of location that may take multiple service requests to iterate over. </returns>
+        public IEnumerable<LocationData> ListAvailableLocations()
```

Comment:
Your writer needs to add a CancellationToken here so that it matches the Async version